corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -664,9 +664,16 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                     ? `${headerWords.join(' ')} <b>${lastChar}</b>`
                     : header;
             };
+            let description = this.options.description;
+            let title = this.options.title;
+            if (this.options.sidebarEnabled && this.options.collection.queryResponse) {
+                description = this.options.collection.queryResponse.description;
+                title = this.options.collection.queryResponse.title;
+            }
             return {
                 startPage: paginateItems.startPage,
-                title: this.options.title,
+                title: title.trim(),
+                description: description === undefined ? "" : DOMPurify.sanitize(markdown.render(description.trim())),
```

Comment:
curious why we are only doing `markdown.render()` for `description` and not `title` ?